api/search.go

```diff
@@ -209,14 +237,20 @@ func CreateRequests(w http.ResponseWriter, req *http.Request, cfg *config.Config
 		return "", nil, nil
 	}
 
+	cdids, cdidErr := parseCDID(ctx, params)
+	if cdidErr != nil {
+		http.Error(w, cdidErr.Error(), http.StatusBadRequest)
```

Comment:
[2/3] could be argued that this is where the variable error message should be constructed

Reply:
Made it static, but kept the warn in `parseCDID` as part of the warn message uses a result of `paramGet` which is used within `parseCDID`. Unless we also pass that result back?